templates/books-show.html

```diff
@@ -0,0 +1,46 @@
+<h2>{{booksShowCtrl.livros.title}}</h2>
+
+<hr>
+
+<div class="row">
+	<div class="col-lg-6 col-md-6 col-sm-6 col-xs-6">
+		<img ng-src="{{booksShowCtrl.livros.image}}" class="img-responsive">
+	</div>
+
+	<div class="col-lg-6 col-md-6 col-sm-6 col-xs-6">
+		<p>Author: {{booksShowCtrl.livros.author}}</p>
+		<p> Release Date: {{booksShowCtrl.livros.releaseDate}}</p>
+
+		<br>
+
+		<!-- edit button -->
+		<a ng-click="editForm=!editForm" class="btn btn-default">Edit book</a>
+
+		<!-- delete button -->
+		<a ng-click="booksShowCtrl.deleteLivro(booksShowCtrl.livros)" class="btn btn-default">Delete book</a>
+
+		<!-- edit form -->
+		<div ng-show="editForm">
+
+			<br>
+
+	    <form ng-show="editForm" ng-submit="booksShowCtrl.updateBook(booksShowCtrl.livros)">
+	    <!-- input field for title -->
+	      <div class="form-group">
+	        <input type="text" ng-model="booksShowCtrl.livros.title" class="form-control" placeholder="Title">
```

Comment:
You should use labels rather than placeholders to explain what inputs are for. See https://www.nngroup.com/articles/form-design-placeholders/